insertBaseData.sql

```diff
@@ -576,3 +576,4 @@ INSERT INTO adminSettings ([key], value) VALUES ('HELP_LINK_URL', 'http://google
 INSERT INTO adminSettings ([key], value) VALUES ('CONTACT_EMAIL', 'team-anet@dds.mil');
 INSERT INTO adminsettings ([key], value) VALUES ('DAILY_ROLLUP_MAX_REPORT_AGE_DAYS', '14');
 INSERT INTO adminsettings ([key], value) VALUES ('EXTERNAL_DOCUMENTATION_LINK_URL', '');
+INSERT INTO adminsettings ([key], value) VALUES ('GENERAL_BANNER_TEXT', '');
```

Comment:
Is this required for the setting to appear in the GUI? I do not know this part of the code well, but I remember hearing that it is. If that is the case, we need to add something to the migrations to make an empty setting if none exists.